src/main.ts

```diff
@@ -99,21 +98,58 @@ async function createTestAPI() {
         console.log("Creating a new load test '"+testId+"' ");
         console.log("Successfully created load test "+testId);
     }
-    else 
+    else{
         console.log("Test '"+ testId +"' already exists");
+        // test script will anyway be updated by the ado in later steps, this will be error if the test script is not present in the test.
```

Comment:
update the comment to mention GH actions instead

Reply:
comments yeah ill change